crates/wasm-smith/src/core.rs

```diff
@@ -199,6 +198,16 @@ impl Module {
             type_size: 0,
         }
     }
+
+    fn import_count(&self) -> usize {
+        self.imports
+            .iter()
+            .filter(|imp| match imp.entity_type {
+                EntityType::Func(_, _) => true,
+                _ => false,
+            })
+            .count()
+    }
```

Comment:
Is this actually used anywhere? I don't see anything in this commit.

Either way, it should be named `imported_func_count` or something like that since it is only counting function imports and not all imports.